src/assets/components/Articles.jsx

```diff
@@ -0,0 +1,38 @@
+import { useState, useEffect } from "react";
+import { getArticles } from "../../utils";
+
+export default function Articles() {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    getArticles().then((response) => {
+      setArticles(response.articles)
+    }, [])
```

Comment:
Do we need this empty array? We only need that for the useEffect second argument.